refactor(Companion): Use QHttp server (#624)
cpeditor/cpeditor: CHANGELOG.md

### Fixed

> -   Fixed an issue where in some cases CP Editor could not get full data from Google Chrome browsers. (#345 and #624)

Reviewer: ```suggestion
-   Fixed an issue that in some cases CP Editor could not get full data from Competitive Companion. (#345 and #624)
```

We have no evidence that it only happens in Chrome.
